src/Microsoft.DotNet.Interactive.Tests/LanguageKernelTests.cs

```diff
@@ -865,6 +865,23 @@ public async Task it_returns_a_result_for_a_if_expressions(Language language, st
             .Be(25);
     }
 
+    [Theory]
+    [InlineData(Language.CSharp, "20")]
+    [InlineData(Language.FSharp, "20")]
+    [InlineData(Language.CSharp, "new int[] {1,2,3}")]
+    [InlineData(Language.FSharp, "[|1;2;3|]")]
+    public async Task it_returns_a_result_with_specified_mimetype(Language language, string expression)
+    {
+        var kernel = CreateKernel(language);
+
+        var command = new SubmitCode(expression, mimeTypes: new []{JsonFormatter.MimeType});
```

Comment:
I find the API a little strange - for example the specified mime types apply only to `ReturnValueProduced` but not to other events...